.github/workflows/release.yaml

```diff
@@ -86,49 +227,73 @@ jobs:
-          call setenv.cmd
-          msbuild -p:Configuration=Release,Platform=Win32 -m DDS_TAOv2.sln
+          Expand-Archive -LiteralPath '.\OutputOpenDDS\OpenDDS_Headers_3.25.zip' -DestinationPath .\third_party\OpenDDS -Force
 
-      - name: Move OpenDDS x86
+      - name: Move OpenDDS x64
         shell: cmd
         run: |
-          cd third_party\OpenDDS
-          xcopy .\lib\*.dll ..\..\Output\x86\
-          xcopy .\lib\*.pdb ..\..\Output\pdb32\
-          xcopy .\lib\*.lib ..\..\Output\lib32\
-          xcopy .\bin\*.exe ..\..\Output\bin32\
-          xcopy .\bin\*.pdb ..\..\Output\pdb32\
-          xcopy .\ACE_wrappers\lib\*.dll ..\..\Output\x86\
-          xcopy .\ACE_wrappers\lib\*.pdb ..\..\Output\pdb32\
-          xcopy .\ACE_wrappers\lib\*.lib ..\..\Output\lib32\
-          xcopy .\ACE_wrappers\bin\*.exe ..\..\Output\bin32\
-          xcopy .\ACE_wrappers\bin\*.pdb ..\..\Output\pdb32\
+           xcopy .\Output\bin32\*.* .\third_party\OpenDDS\bin\
+           xcopy .\Output\x86\*.dll .\third_party\OpenDDS\lib\
+           xcopy .\Output\bin32\*.* .\third_party\OpenDDS\ACE_wrappers\bin\
+           xcopy .\Output\x86\*.dll .\third_party\OpenDDS\ACE_wrappers\lib\
+
+    #  - name: Build OpenDDS x64
+    #    shell: cmd
+    #    run: |
+    #      cd third_party\OpenDDS
+    #      call setenv.cmd
+    #      msbuild -p:Configuration=Release,Platform=x64 -m DDS_TAOv2.sln
+    #
+    #  - name: Move OpenDDS x64
+    #    shell: cmd
+    #    run: |
+    #      cd third_party\OpenDDS
+    #      xcopy .\lib\*.dll ..\..\Output\x64\
+    #      xcopy .\lib\*.pdb ..\..\Output\pdb64\
+    #      xcopy .\lib\*.lib ..\..\Output\lib64\
+    #      xcopy .\bin\*.exe ..\..\Output\bin64\
+    #      xcopy .\bin\*.pdb ..\..\Output\pdb64\
+    #      xcopy .\ACE_wrappers\lib\*.dll ..\..\Output\x64\
+    #      xcopy .\ACE_wrappers\lib\*.pdb ..\..\Output\pdb64\
+    #      xcopy .\ACE_wrappers\lib\*.lib ..\..\Output\lib64\
+    #      xcopy .\ACE_wrappers\bin\*.exe ..\..\Output\bin64\
+    #      xcopy .\ACE_wrappers\bin\*.pdb ..\..\Output\pdb64\
+    #
+    #  - name: Build OpenDDS x86
+    #    shell: cmd
+    #    run: |
+    #      cd third_party\OpenDDS
+    #      call setenv.cmd
+    #      msbuild -p:Configuration=Release,Platform=Win32 -m DDS_TAOv2.sln
+    #
+    #  - name: Move OpenDDS x86
+    #    shell: cmd
+    #    run: |
+    #      cd third_party\OpenDDS
+    #      xcopy .\lib\*.dll ..\..\Output\x86\
+    #      xcopy .\lib\*.pdb ..\..\Output\pdb32\
+    #      xcopy .\lib\*.lib ..\..\Output\lib32\
+    #      xcopy .\bin\*.exe ..\..\Output\bin32\
+    #      xcopy .\bin\*.pdb ..\..\Output\pdb32\
+    #      xcopy .\ACE_wrappers\lib\*.dll ..\..\Output\x86\
+    #      xcopy .\ACE_wrappers\lib\*.pdb ..\..\Output\pdb32\
+    #      xcopy .\ACE_wrappers\lib\*.lib ..\..\Output\lib32\
+    #      xcopy .\ACE_wrappers\bin\*.exe ..\..\Output\bin32\
+    #      xcopy .\ACE_wrappers\bin\*.pdb ..\..\Output\pdb32\
```

Comment:
Clean code

Reply:
Leaved in workflow till completed at least a first release cycle